### PR TITLE
fix: release-plz release only runs on "chore: release" PRs

### PR DESCRIPTION
**Pull request by danking**
I also updated to use the new release-pr command to generate PRs in a race-free way.

### CODE REVIEW

File: .github/workflows/release-plz.yml [hunk 1]
```diff
@@ -10,25 +10,50 @@ on:
...
 
 jobs:
-  release-plz:
-    name: Release-plz
+
+  # Release unpublished packages.
+  release-plz-release:
+    name: Release-plz release
     runs-on: ubuntu-latest
+    if: startsWith(github.event.head_commit.message, "chore: release")
```
Comment (a10y) [inline .github/workflows/release-plz.yml:18]: can you have `if` up here or does it need to live in `steps` ?

Comment (a10y) [reply .github/workflows/release-plz.yml:18]: ah nvm it indeed can live here 👍

**Review by a10y — APPROVED** [review]

File: .github/workflows/release-plz.yml (hunk 1 above)
Comment (danking) [reply .github/workflows/release-plz.yml:18]: Rats you were right https://github.com/spiraldb/vortex/actions/runs/11636435105/workflow

Comment (danking) [reply .github/workflows/release-plz.yml:18]: Ah I am maybe just missing ${{ }}?

**Pull request merged**